### PR TITLE
Adds imaginary friends for the mentors.

### PR DESCRIPTION
**Pull request by DominikPanic**
Promises are to be fulfilled.
:cl: LaKiller8
tweak: Newer players may start having illussions of a friendly ghost guiding them. This is an imaginary friend controlled by one of our fellow mentors.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
This is cool and neat enough to warrant a changelog.
:heart:

**Comment by DominikPanic** [comment]
> 
> 
> This is cool and neat enough to warrant a changelog.
> heart

Added, feel free to edit if you'd like a different wording, felt like a slight joke was approriate.

File: code/modules/mob/camera/imaginary_friend.dm [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+	gender = client.prefs.gender
+	human_image = get_flat_human_icon(null, SSjob.GetJobType(/datum/job/other/spatial_agent), client.prefs)
+
+
+/mob/camera/imaginary_friend/proc/Show()
+	if(!client)
+		return
+
+	if(owner.client)
+		owner.client.images.Remove(current_image)
```
Comment (Rohesie) [inline code/modules/mob/camera/imaginary_friend.dm]:
```suggestion
		owner.client?.images.Remove(current_image)
```

File: code/modules/mob/camera/imaginary_friend.dm [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+	client.images |= current_image
+
+
+/mob/camera/imaginary_friend/Destroy()
+	if(owner.client)
+		owner.client.images.Remove(human_image)
+
+	if(client)
+		client.images.Remove(human_image)
```
Comment (Rohesie) [inline code/modules/mob/camera/imaginary_friend.dm]:
```suggestion
		client?.images.Remove(human_image)
```

**Comment by Rohesie** [comment]
Perfect.

**Pull request merged**